src/components/chips/js/chipsDirective.js

```diff
@@ -108,7 +108,7 @@
    * @param {expression=} md-on-add An expression which will be called when a chip has been
    *    added.
    * @param {expression=} md-on-remove An expression which will be called when a chip has been
-   *    removed.
+   *    removed with $chip, $index and $event available as parameters.
```

Comment:
Can you please wrap these parameter names with backticks? It will help make the docs more clear.